src/systems/audio-system.js

```diff
@@ -76,12 +115,15 @@ export class AudioSystem {
     this.analyserLevels = new Uint8Array(this.outboundAnalyser.fftSize);
     this.outboundGainNode.connect(this.outboundAnalyser);
     this.outboundAnalyser.connect(this.mediaStreamDestinationNode);
+    let alreadyExecuting = false;
 
     /**
      * Chrome and Safari will start Audio contexts in a "suspended" state.
      * A user interaction (touch/mouse event) is needed in order to resume the AudioContext.
      */
     const resume = () => {
+      if (alreadyExecuting) return;
+      alreadyExecuting = true;
```

Comment:
Remove this. This causes a regression that sometimes the enableChromeAEC is not executed because the audioContext is still not in the running state when the callback from the setTimeout is executed.